services/text_to_speech/v1-corpus-builder.js

```diff
@@ -47,160 +48,193 @@ module.exports = function (RED) {
     sEndpoint = service.url;
   }
 
-  function executeCreateCustomisation(node, tts, params, msg) {
-    tts.createCustomization(params, function (err, response) {
-      node.status({});
-      if (err) {
-        payloadutils.reportError(node, msg, err);
+  function parseResponseFor(msg, response, field) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)